conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -173,13 +173,18 @@ types:
           auth: optional<AuthType>
           args: list<ArgumentDefinition>
           returns: optional<Type>
+          errors: list<EndpointError>
```

Comment:
List makes sense because the same error could be listed twice in a set with slightly different docs, so we’ll have to validate uniqueness ourselves anyhow

Reply:
Oh true. I'll add that validation.